app/models/shipit/pull_request.rb

```diff
@@ -171,6 +172,11 @@ def github_pull_request=(github_pull_request)
       self.head = find_or_create_commit_from_github_by_sha!(github_pull_request.head.sha, detached: true)
     end
 
+    def merge_message
+      return title unless merge_requested_by
+      "#{title}\n\nMerge-Requested-By: #{merge_requested_by.login}\n"
```

Comment:
No reason. I just felt like respecting the "header" format: e.g.: https://github.com/torvalds/linux/commit/912964eacb111551db73429719eb5fadcab0ff8a